openreview/venue_request/venue_request.py

```diff
@@ -1145,6 +1146,74 @@ def setup_venue_recruitment(self):
                 }
             ))
 
+        recruitment_status_content = {
+            'title': {
+                'value': 'Recruitment Status',
+                'required': True,
+                'order': 1
+            },
+            'invited': {
+                'value-regex': '.*',
+                'description': 'No. of users invited',
+                'required': True,
+                'markdown': True,
+                'order': 2
+            },
+            'already_invited': {
+                'value-regex': '[\\S\\s]{0,100000}',
```

Comment:
if this is a list then it should be `values-regex: '.*'` so the value is an array of strings instead of an string

Reply:
It's a dictionary actually, do you want me to combine all the lists?

Reply:
It's a dictionary ([here](https://github.com/openreview/openreview-py/blob/717c9e7927d5d019a0161fcd197cc47f210ee230/openreview/conference/builder.py#L1251)). It maps `group_id` to the list of profiles. ([here](https://github.com/openreview/openreview-py/blob/717c9e7927d5d019a0161fcd197cc47f210ee230/openreview/conference/builder.py#L1376))

Reply:
I see, because the user could come from different groups. 

How does the UI render this value? the field type should `value-json` instead of `value-regex`.